templates/Gemfile.erb

```diff
@@ -30,7 +30,7 @@ end
 
 group :development, :test do
   gem "bundler-audit", require: false
-  gem "byebug"
+  gem "pry-byebug"
```

Comment:
:+1: will change.. it's likely I missed that on many of the recent round of open PRs